egs/babel/s5d/conf/mfcc_hires_bp.conf

```diff
@@ -0,0 +1,10 @@
+# config for high-resolution MFCC features, intended for neural network training.
+# Note: we keep all cepstra, so it has the same info as filterbank features,
+# but MFCC is more easily compressible (because less correlated) which is why
+# we prefer this method.
+--use-energy=false   # use average of log energy, not energy.
+--sample-frequency=8000 #  Switchboard is sampled at 8kHz
```

Comment:
I'm concerned that the configuration here is only slightly different from our regular hires configuration (28 vs. 40 mel bins), and isn't it a bit wasteful to separately dump MFCCs with a slightly different configuration?